src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/EventTypeTest.java

```diff
@@ -171,6 +171,7 @@ public class EventTypeTest {
             "eventsUpdates\n" +
             "updateHearingType\n" +
             "updateIncompleteApplication\n" +
+            "updateListingRequirements\n" +
```

Comment:
What is this test?

Reply:
checks all these event types are in the CCD def file